### PR TITLE
MCLOUD-2789: [Cloud Docker] Add custom ES plugins

### PR DESCRIPTION
**Pull request by oshmyheliuk**
## Purpose of this pull request

This pull request (PR) will add information about elasticsearch plugins in the docker container.

## Affected DevDocs pages

-  https://devdocs.magento.com/cloud/docker/docker-containers-service.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Review by BarnyShergold — APPROVED** [review]

File: src/cloud/docker/docker-containers.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ The following table shows the options to customize service container configurati
...
 | [db] | MariaDB or MySQL<br>     | `--db`, `--db-image` (MySQL)<br>`--expose-db-port`<br>`--db-increment`<br>`--db-offset`<br>`--with-entrypoint`<br>`--with-mariadb-config`|10.0, 10.1, 10.2<br>5.6, 5.7 | Use the increment and offset options to customize the [auto-increment settings][Using AUTO_INCREMENT] for replication.<br><br>Use the `--with-entrypoint` and `--with-mariadb-config` options to automatically configure database directories in the Docker environment<br><br>*Example build commands:*<br>`ece-docker build:compose --db <mariadb-version>`<br>`ece-docker build:compose --db <mysql-version> --db-image`
-| [elasticsearch] | Elasticsearch | `--es`<br>`--es-env-var`<br>`--no-es` | 1.7, 2.4, 5.2, 6.5, 6.8, 7.5, 7.6 | Use the options to specify the Elasticsearch version,  customize Elasticsearch configuration options, or to build a Docker environment without Elasticsearch.
+| [elasticsearch] | Elasticsearch | `--es`<br>`--es-env-var`<br>`--no-es` | 1.7, 2.4, 5.2, 6.5, 6.8, 7.5, 7.6, 7.7 | Use the options to specify the Elasticsearch version,  customize Elasticsearch configuration options, or to build a Docker environment without Elasticsearch.
```
Comment (meker12) [inline src/cloud/docker/docker-containers.md:46]: Should we add a note that about the supported versions when deploying to Cloud and point to the version compatibility table?

File: src/cloud/docker/docker-containers-service.md [hunk 2]
```diff
@@ -75,6 +75,19 @@ See [Important Elasticsearch configuration][] in the Elasticsearch documentation
 {:.bs-callout-info}
 If your Cloud project uses Magento version 2.3.5 or earlier with MySQL search, add the `--no-es` option to skip the Elasticsearch container configuration when you generate the Docker Compose configuration file: `ece-docker build:compose --no-es`.
 
+### Elasticsearch plugins
+
+Plugins `analysis-icu` and `analysis-phonetic` are installed by default and can not be skipped.
+Elasticsearch plugins configured in `.magento/services.yaml` will be automatically installed in elasticsearch container starting from version 6.5.
+You can change list of plugins in `docker-compose.yaml` file:
```
Comment (meker12) [inline src/cloud/docker/docker-containers-service.md:82]:
```suggestion
The `analysis-icu` and `analysis-phonetic` plugins are installed by default and can not be skipped.
If you use Elasticsearch 6.5 and later, the Elasticsearch plugins configured in `.magento/services.yaml` are installed automatically.

You can change the list of plugins in `docker-compose.yaml` file:
```

**Review by meker12 — CHANGES_REQUESTED** [review]

File: src/cloud/docker/docker-containers-service.md (hunk 2 above)
Comment (oshmyheliuk) [reply src/cloud/docker/docker-containers-service.md:82]: The plugin list will be automatically generated from plugins mentioned in `.magento/services.yaml` but it's possible to edit the generated `docker-compose.yaml` file to change plugin list.
So, the command will generate the next code in `docker-compose.yaml`
```
services:
    elasticsearch:
        environment:
          - 'ES_PLUGINS=analysis-stempel analysis-nori'
```
And before running `docker-compose up -d` it's possible to edit this environment variable manually for adding/removing required plugins.

**Review by meker12 — APPROVED** [review]

**Pull request merged**